app/main.ipynb

```diff
@@ -563,25 +818,52 @@
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T14:11:19.696302Z",
-     "start_time": "2023-07-03T14:11:19.678876Z"
+     "end_time": "2024-11-29T22:47:26.884451Z",
+     "start_time": "2024-11-29T22:47:26.872846Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[0, 2, 0, 4, 0]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 6
   },
   {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-11-29T22:57:25.931992Z",
+     "start_time": "2024-11-29T22:57:25.917525Z"
+    }
+   },
    "cell_type": "code",
-   "execution_count": 30,
-   "outputs": [],
    "source": [
-    "# write your numpy code here"
+    "# write your numpy code here\n",
+    "np_lst = np.arange(1, 6, dtype=np.int8)\n",
+    "np_mask = np.array(mask, dtype=bool)\n",
+    "\n",
+    "np.where(np_mask, 0, np_lst)\n"
```

Comment:
In Task 7.1, the numpy code using `np.where` does not modify the original array `np_lst`. To achieve the same effect as the Python code, you should assign the result back to `np_lst` or use in-place modification.